services/github-service-impl/pom.xml

```diff
@@ -1,51 +1,108 @@
-   <packaging>ejb</packaging>
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.kontinuity.catapult</groupId>
+        <artifactId>catapult-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>catapult-service-github-impl</artifactId>
+    <packaging>ejb</packaging>
 
-   <dependencies>
-      <dependency>
-         <groupId>org.kohsuke</groupId>
-         <artifactId>github-api</artifactId>
-      </dependency>
-      <dependency>
-         <groupId>com.squareup.okhttp</groupId>
-         <artifactId>okhttp-urlconnection</artifactId>
-      </dependency>
-      <dependency>
-      	<groupId>org.kontinuity.catapult</groupId>
-      	<artifactId>catapult-service-github-api</artifactId>
-      </dependency>
-   </dependencies>
-   <build>
-      <plugins>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-ejb-plugin</artifactId>
-            <configuration>
-               <ejbVersion>3.0</ejbVersion>
-            </configuration>
-         </plugin>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-jar-plugin</artifactId>
-            <version>2.6</version>
-            <executions>
-               <execution>
-                  <goals>
-                     <goal>test-jar</goal>
-                  </goals>
-               </execution>
-            </executions>
-         </plugin>
-      </plugins>
-   </build>
+    <dependencies>
+        <dependency>
+            <groupId>org.kohsuke</groupId>
+            <artifactId>github-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp</groupId>
+            <artifactId>okhttp-urlconnection</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.kontinuity.catapult</groupId>
+            <artifactId>catapult-service-github-api</artifactId>
+        </dependency>
+
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-ejb-plugin</artifactId>
+                <configuration>
+                    <ejbVersion>3.0</ejbVersion>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
```

Comment:
Code organization: the "tests" module is for tests against the full EAR (or any other artifacts) as built by Maven.  If you want to run tests with microdeployments (as you define them using ShrinkWrap), then you can do them here in this module.  It's about the level of integration you wanna hit. :)